node/src/actors/json_rpc/json_rpc_methods.rs

```diff
@@ -232,8 +232,11 @@ pub fn jsonrpc_io_handler(
                 };
 
                 match method_name.as_str() {
-                    "newBlocks" => {
```

Comment:
`newBlocks` is more clear than `blocks`. Unless you can now receive old blocks?

Reply:
Using your same logic, if the node can only notify new stuff as it comes, there's no point in tagging anything as "new" — we simply know it is.

Reply:
Still, I would prefer to leave `newBlocks` as an alias to `blocks` to avoid breaking changes.